test/fixtures_stage_based_messaging.js

```diff
@@ -0,0 +1,867 @@
+
+module.exports = function() {
+    return [
+
+        // 0: get active subscriptions for cb245673-aa41-4302-ac47-00000001001
+        {
+            "request": {
+                "url": 'http://sbm.localhost:8001/api/v1/subscriptions/',
+                "method": 'GET',
+                "params": {
+                    "identity": 'cb245673-aa41-4302-ac47-00000001001',
+                    "active": 'true'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 0,
+                    "next": null,
+                    "previous": null,
+                    "results": []
+                }
+            }
+        },
+
+        // 1: get active subscriptions for cb245673-aa41-4302-ac47-00000001002
+        {
+            "request": {
+                "url": 'http://sbm.localhost:8001/api/v1/subscriptions/',
+                "method": 'GET',
+                "params": {
+                    "identity": 'cb245673-aa41-4302-ac47-00000001002',
+                    "active": 'true'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [
+                        {
+                            'url': 'http://sbm.localhost:8001/api/v1/subscriptions/51fcca25-2e85-4c44-subscription-1002',
+                            'id': '51fcca25-2e85-4c44-subscription-1002',
+                            'version': 1,
+                            'identity': 'cb245673-aa41-4302-ac47-00000001002',
+                            'messageset': 1,
+                            'next_sequence_number': 1,
+                            'lang': "en",
```

Comment:
Hmm. This fixture looks suspicious since `lang` is a 2-char string, not 6-char.  Let's make it look as close to reality as possible.